### ISSUE TITLE
Broken with postgres adaptor

### ISSUE THREAD

**Issue by renequez** [boomworks/payload-plugin-author-fields#27]
Using payload with the postgres adapter breaks this plugin because in https://github.com/boomworks/payload-plugin-author-fields/blob/5da3a005db53c45870458646e0195f89ad3aee2d/src/DisplayOnlyField/DisplayOnlyField.tsx#L52 it tries to fetch by `_id` instead of `id`:

```bash
APIError: Cannot find field for path at _id
    at getTableColumnFromPath (/payload-postgres/node_modules/@payloadcms/db-postgres/src/queries/getTableColumnFromPath.ts:439:9)
    at parseParams (/node_modules/@payloadcms/db-postgres/src/queries/parseParams.ts:88:43)
    at buildAndOrConditions (/node_modules/@payloadcms/db-postgres/src/queries/buildAndOrConditions.ts:38:39)
    at parseParams (/node_modules/@payloadcms/db-postgres/src/queries/parseParams.ts:55:61)
    at buildQuery (/node_modules/@payloadcms/db-postgres/src/queries/buildQuery.ts:103:30)
    at find (/node_modules/@payloadcms/db-postgres/src/find/findMany.ts:44:80)
    at Object.find (/node_modules/@payloadcms/db-postgres/src/find.ts:26:18)
    at find (/node_modules/payload/src/collections/operations/find.ts:144:33)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at findHandler (/node_modules/payload/src/collections/requestHandlers/find.ts:30:20)
```

**Comment by lindsayevans** [comment]
Thanks for reporting @renequez - I've just published v1.1.8 which should fix this issue

**Comment by WillFelisberto** [comment]
Hello @lindsayevans , it's still happening in v1.1.8.
